### PR TITLE
Fix BigRNN table on website

### PR DESCRIPTION
**Pull request by leezu**
## Description ##
@eric-haibin-lin 

### Essentials ###
- [X] Changes are complete (i.e. I finished coding on this PR)
- [X] All changes have test coverage
- [X] Code is well-documented

### Changes ###
- [X] Fix BigRNN table on website

### CODE REVIEW

**Comment by mli** [comment]
Job PR-257/1 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-257/1/index.html

File: scripts/language_model/language_model.rst [hunk 1]
```diff
@@ -156,20 +156,35 @@ The key features used to reproduce the results for pre-trained models are listed
...
+| Batch size      | 256           |
++-----------------+---------------+
+| Graident clip   | 10.0          |
++-----------------+---------------+
 | Test perplexity | 44.05         |
-| Num epochs      | 48            |
++-----------------+---------------+
+| Num epochs      | 48            | 
++-----------------+---------------+
+| Tied            | True          |
```
Comment (eric-haibin-lin) [inline scripts/language_model/language_model.rst]: It's not tied. could you remove this line?

File: scripts/language_model/language_model.rst [hunk 2]
```diff
@@ -156,20 +156,35 @@ The key features used to reproduce the results for pre-trained models are listed
 
 The dataset used for training the models is Google's 1 billion words dataset.
 
++-----------------+---------------+
 | Model           | LSTM-2048-512 |
-| --------------- | ------------- |
++=================+===============+
+| Mode            | LSTM          |
```
Comment (eric-haibin-lin) [inline scripts/language_model/language_model.rst]: The cell used is LSTMP, which is different from LSTM cells. Would make the mode "LSTMP" less confusing?

**Comment by mli** [comment]
Job PR-257/2 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-257/2/index.html

**Comment by mli** [comment]
Job PR-257/3 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-257/3/index.html

**Pull request merged**